NOTES.md

```diff
@@ -1,3 +1,191 @@
+[release blog post][1.16-blog] for more details.
+
+[1.16-blog]: http://rethinkdb.com/blog/1.16-release/
+
+There may or may not be API-breaking changes in this release.
+
+## New Features ##
+
+* ReQL admin: a new cluster management and monitoring API (#2169)
+  * Added a system database named `rethinkdb`
+    * The `cluster_config` table contains cluster-wide settings (#2924)
+    * The `current_issues` table lists issues affecting the cluster and suggests solutions (#2864, #3258)
+    * The `db_config` table allows listing and renaming databases (#151, #2871)
+    * The `jobs` table lists running jobs (#3115)
+    * The `logs` table for accessing the server logs (#2884)
+    * The `server_config` table for managing servers (#2873)
+    * The `server_status` table contains information about the servers in the cluster (#2923, #3270)
+    * The `stats` table contains real-time statistics about the server (#2885)
+    * The `table_config` table allows listing, creating, modifying and dropping tables (#2870)
+    * The `table_status` table contains extra information about each table (#2983, #3269)
+    * The hidden `_debug_table_status` table contains internal debugging information (#2901)
+    * The hidden `_debug_stats` table contains internal statistics (#3385)
+    * The `identifier_format` optional argument to `table` switches between table names (when set to "name") and table identifiers (when set to "uuid") (#3266)
+  * Improved cluster management
+    * Servers can now be associated with multiple tags using the `--server-tag` flag or by updating the `server_config` table (#2856)
+    * Removed the datacenter abstraction and changed the arguments to `tableCreate` (#2876)
+    * Added a `reconfigure` command to change the replication and sharding settings of a table (#2932)
+    * Added a `rebalance` command to even out the size of a table's shards (#2981)
+    * Added a `config` command for tables and databases as an alias for the corresponding row in `db_config` or `table_config`
+    * Added a `status` command for tables and databases as an alias for the corresponding row in `db_status` or `table_satus`
+    * Redundant information has been removed from the `/ajax` endpoint (#2878, #2879)
+    * Automatically migrate data directories to the new semilattice configuration format (#2869)
+    * The stats now contains the number of client connections (#2989)
+    * Added more information to the return value of `db_create`, `db_drop`, `table_create` and `table_drop` (#3001)
+    * Changed how `durability` and `write_acks` are configured (#3066)
+    * Now allow changing the cache size at runtime (#3166)
+    * Now expose the command line of the server (#3180)
+    * Improved the scalability for large table creation and reconfiguration in large clusters (#3198)
+    * Added a new UI for table configuration (#3229)
+    * Now allow over-sharding tables when there are not enough documents (#3271)
```

Comment:
I'm not sure if the term "over-sharding" is appropriate here. We usually use that term to mean sharding to more shards than there are servers, but this is about the number of documents in the table.

Maybe
"Allow sharding tables even when there are not enough documents to determine the key distribution (#3271)"